app/components/widgets/StreamBoss.vue.ts

```diff
@@ -22,6 +22,8 @@ import TestButtons from './TestButtons.vue';
 })
 export default class StreamBoss extends WidgetSettings<IStreamBossData, StreamBossService> {
 
+  tab = 'goal';
```

Comment:
what's this doing here? i don't see it map to anything in the template

Reply:
This is a default tab that is using in `WidgetWindow.vue` . WidgetWindow is deprecated now but WidgetEditor uses `source` tab as default which actually shouldn't be a default one since not so many users really need it. We must decide how we determine a default tab. It looks like it can just be the first item in `navItems` but it worth to check. If it will work we can delete `tab` here.

Reply:
ah ok, yea defaulting to source was mostly just a placeholder but i think defaulting to the first `navItem` should be fine